test/unit/specs/store/user.spec.js

```diff
@@ -0,0 +1,28 @@
+import * as user from '~/store/user'
+import { v4 as uuidv4 } from 'uuid'
+
+jest.mock('uuid', () => ({ v4: jest.fn() }))
+
+describe('user store', () => {
+  beforeEach(() => {
+    process.server = false
+    process.client = false
+  })
+
+  afterAll(() => {
+    // avoid polluting other tests that might be affected by these flags
```

Comment:
Nice cleanup and comment of why it's necessary, especially for future use and copying :)